src/main/java/com/box/sdk/BoxFile.java

```diff
@@ -29,7 +29,7 @@
     public static final String[] ALL_FIELDS = {"type", "id", "sequence_id", "etag", "sha1", "name", "description",
         "size", "path_collection", "created_at", "modified_at", "trashed_at", "purged_at", "content_created_at",
         "content_modified_at", "created_by", "modified_by", "owned_by", "shared_link", "parent", "item_status",
-        "version_number", "comment_count", "permissions", "tags", "lock", "extension", "is_package"};
+        "version_number", "comment_count", "permissions", "tags", "lock", "extension", "is_package", "file_version"};
```

Comment:
Can you add the `file_version` field to `BoxItem.ALL_FIELDS` constant as well? `BoxItem.ALL_FIELDS` is just a superset of all the fields that can be used with files or folders.